bff/cron/active_users_test.go

```
@@ -0,0 +1,121 @@
+	}
+	return 0
+}
+
+func TestActiveUsers24hRefresherCountsRollingWindow(t *testing.T) {
+	mr, client := newTestRedis(t)
+	gauge := newTestGauge()
+	refresher := NewActiveUsers24hRefresher(client, gauge, nopLogger{})
+	now := time.Date(2026, 6, 16, 10, 15, 0, 0, time.Local)
+
+	// Both boundaries are included: [previous day 10:15, today 10:15).
+	mr.PfAdd(ActiveUsersBucketKey(now.Add(-24*time.Hour)), "stu-1")
```

golangci-lint (2.12.2)

[error] Error return value of `mr.PfAdd` is not checked

(errcheck)